src/index.html

```diff
@@ -516,43 +518,201 @@ <h3 slot="title">ArcGIS Online Sign In and Sign Up</h3>
+              <calcite-tree lines>
+                <calcite-tree-item> Child 1 </calcite-tree-item>
+
+                <calcite-tree-item>
+                  Child 2
+
+                  <calcite-tree slot="children">
+                    <calcite-tree-item> Grandchild 1 </calcite-tree-item>
+
+                    <calcite-tree-item>
+                      Grandchild 2
+                      <calcite-tree slot="children">
+                        <calcite-tree-item> Great-Grandchild 1 </calcite-tree-item>
+                        <calcite-tree-item> Great-Grandchild 2 </calcite-tree-item>
+                      </calcite-tree>
+                    </calcite-tree-item>
+                  </calcite-tree>
+                </calcite-tree-item>
+                <calcite-tree-item> Child 3 </calcite-tree-item>
+              </calcite-tree>
+            </calcite-accordion-item>
+          </calcite-accordion>
+        </div>
+
+        <div>
+          <calcite-card>
+            <img slot="thumbnail" alt="Sample image alt" src="https://placeimg.com/200/100/nature" />
+            <h3 slot="title">Untitled experience</h3>
+            <span slot="subtitle">Subtext</span>
+            <calcite-button slot="footer-leading" width="full">Go</calcite-button>
+          </calcite-card>
+        </div>
+
+        <div>
+          <calcite-date-picker scale="m" value="2020-11-27" active></calcite-date-picker>
+        </div>
+
+        <div>
+          <calcite-tabs active>
+            <calcite-tab-nav slot="tab-nav">
+              <calcite-tab-title active>Tab 1 Title</calcite-tab-title>
+              <calcite-tab-title>Tab 2 Title</calcite-tab-title>
+              <calcite-tab-title>Tab 3 Title</calcite-tab-title>
+              <calcite-tab-title>Tab 4 Title</calcite-tab-title>
+            </calcite-tab-nav>
+
+            <calcite-tab active>
+              <calcite-loader active text="loading tab 1&hellip;" type="indeterminate"></calcite-loader>
+            </calcite-tab>
+            <calcite-tab>
+              <calcite-loader active text="loading tab 2&hellip;" type="indeterminate"></calcite-loader>
+            </calcite-tab>
+            <calcite-tab>
+              <calcite-loader active text="loading tab 3&hellip;" type="indeterminate"></calcite-loader>
+            </calcite-tab>
+            <calcite-tab>
+              <calcite-loader active text="loading tab 4&hellip;" type="indeterminate"></calcite-loader>
+            </calcite-tab>
+          </calcite-tabs>
+        </div>
       </div>
     </div>
+
+    <script>
+      const input = document.getElementById("search");
+      const flex = document.querySelectorAll("#parent-flexbox div");
+      console.log(flex);
+
+      input.addEventListener("keyup", (event) => {
+        const value = event.target.value;
+        flex.forEach((item) => {
+          item.querySelector("a calcite-tile-select[heading]").textContent.toLowerCase().startsWith(value)
```

Comment:
You can access the heading property like:

` item.querySelector("a calcite-tile-select").heading`

You might also want to ensure value is `toLowerCase`.